ui/src/pages/namespace/Explorer/Page/poc/schema/primitives/variable.ts

```diff
@@ -4,11 +4,17 @@ export const Variable = z.string().min(1);
 
 export type VariableType = z.infer<typeof Variable>;
 
-const variableNamespaces = ["query", "loop", "form"] as const;
+const globalVariableNamespaces = ["query", "loop"] as const;
```

Comment:
```suggestion
const contextVariableNamespaces = ["query", "loop"] as const;
```
In line with my other suggestions